Doc/library/tarfile.rst

```diff
@@ -390,7 +426,7 @@ be finalized; only the internally used file object will be closed. See the
    available.
 
 
-.. method:: TarFile.extractall(path=".", members=None, *, numeric_owner=False)
+.. method:: TarFile.extractall(path=".", members=None, *, numeric_owner=False, filter=None)
```

Comment:
Thanks for the review! I'll get to the rest later, but here I'll reply now:

It would need to be `extraction_filter`. The `add` method has a different `filter` argument (which also isn't duplicated in `__init__`).
But I don't think putting it here would make things easier for users.

Reply:
Ah, okay.